src/SecretsMocker/SecretsMocker/Dockerfile

```diff
@@ -1,20 +1,16 @@
-FROM --platform=$BUILDPLATFORM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
-WORKDIR /app
-EXPOSE 80
-
-
 FROM --platform=$BUILDPLATFORM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+
 WORKDIR '/src/SecretsMocker/SecretsMocker'
 COPY . .
 RUN dotnet restore './SecretsMocker.csproj'
 RUN dotnet build './SecretsMocker.csproj' -c Release -o /app/build --no-restore
 
-
 FROM build AS publish
+
 RUN dotnet publish './SecretsMocker.csproj' --no-restore -c Release -o /app/publish /p:UseAppHost=false --self-contained false
```

Comment:
Out of curiosity, why not build for a specific RID? Since you control the platform, you could do something like `--arch $TARGETARCH` and get platform-and-arch-specific builds that would likely have a smaller size?

Reply:
I did try this, but it failed, see https://github.com/LanceMcCarthy/akeyless-web-target/commit/2024301fe4f3ba25204e3787379e2c91d9dc5aff